File: src/_1ProgrammingBasicsWithJavaMay2023/_3ConditionalStatementsAdvanced/_3MoreExercise/_10MultiplyBy2.java
package _1ProgrammingBasicsWithJavaMay2023._3ConditionalStatementsAdvanced._3MoreExercise;

import java.util.Scanner;

public class _10MultiplyBy2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double n = Double.parseDouble(scanner.nextLine());
        double result = 0.0;
        while (n>= 0) {
            result = n*2;
            System.out.printf("Result: %.2f%n",result);
            n = Double.parseDouble(scanner.nextLine());
        }
        System.out.println("Negative number!");
    }
}

File: src/_1ProgrammingBasicsWithJavaScriptNovember2024/_3ConditionalStatementsAdvanced/_3MoreExercise/_10MultiplyBy2.js
function multiplyBy2(numbers) {
    let index = 0;
    let number = Number(numbers[index]);
    while (number >= 0) {
        const result = number * 2;
        console.log(`Result: ${result.toFixed(2)}`);
        index++;
        number = Number(numbers[index]);
    }
    console.log(`Negative number!`);
}